backend/transactions.go

```diff
@@ -193,14 +199,14 @@ func (a *WalletApplication) postTransaction(txObject string) *models.TXHistory {
 
 	if TXSuccessfullyPutOnNetwork {
 		txData := &models.TXHistory{
-			Amount:   tx.Edge.Data.Amount,
-			Sender:   tx.Edge.ObservationEdge.Parents[0].HashReference,
-			Receiver: tx.Edge.ObservationEdge.Parents[1].HashReference,
-			Fee:      tx.Edge.Data.Fee,
-			Hash:     hash,
-			TS:       time.Now().Format("Jan _2 15:04:05"),
-			Status:   "Pending",
-			Failed:   false,
+			Amount:      tx.Edge.Data.Amount,
+			Source:      tx.Edge.ObservationEdge.Parents[0].HashReference,
+			Destination: tx.Edge.ObservationEdge.Parents[1].HashReference,
+			Fee:         tx.Edge.Data.Fee,
+			Hash:        hash,
+			Timestamp:   time.Now().Format(time.RFC3339),
+			Status:      "Pending",
+			Failed:      false,
```

Comment:
It isn't clear to me where `tx` is coming from here but this would be the wrong format for V2. `tx.Edge` doesn't exist.

Reply:
hmmm okay, I'll need to review this.
I'll add it to the list of tasks.